### ISSUE TITLE
comparison is blank

### ISSUE THREAD

**Issue by bcullman** [guyellis/packcomp#11]
getting the following...

```
> packcomp team1/project-1/package.json team2/project-1/package.json

Dependencies:

DevDependencies:
```

**Comment by guyellis** [comment]
What version of Node are you using @bcullman ?

**Comment by guyellis** [comment]
Also @bcullman that might be the output when the dependencies of two `package.json` files are identical in version.

Unfortunately I'm on vacation and my laptop died on me so I won't be able to test this for 2 days...

**Comment by bcullman** [comment]
``` 
> node -v
v8.11.1
```

The 2 package.json files are not in any way identical.

**Comment by guyellis** [comment]
@bcullman how did you install it and how are you running it?

Also can you try and run it and precede the command with: `DEBUG=packcomp:* ` can post the output you are seeing?

E.g.:

```shell
DEBUG=packcomp:* packcomp proj1/package.json proj2/package.json
```

**Comment by bcullman** [comment]
```
> DEBUG=packcomp:* packcomp nui-widgets/package.json ui-assets/package.json
  packcomp:cli titles: [ 'nui-widgets', 'ui-assets' ] +0ms
  packcomp:cli packages: [ '/Users/i847135/Documents/git/github.concur/coreui/nui-widgets/package.json', '/Users/i847135/Documents/git/github.concur/coreui/ui-assets/package.json' ] +3ms

Dependencies:
  packcomp:cli diffObj: { Module: '@concur/nui-widgets-migration-plugin', 'nui-widgets': '^0.2.3', 'ui-assets': 'missing' } +3ms
  packcomp:cli diffObj: { Module: 'classnames', 'nui-widgets': '^2.2.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'debug', 'nui-widgets': '^2.2.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'grouper', 'nui-widgets': '^2.1.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'jest', 'nui-widgets': '^22.4.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'keycode', 'nui-widgets': '^2.1.7', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'moment', 'nui-widgets': '^2.14.1', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'popper.js', 'nui-widgets': '^1.12.9', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'prop-types', 'nui-widgets': '^15.5.8', 'ui-assets': 'missing' } +1ms
  packcomp:cli diffObj: { Module: 'react-addons-create-fragment', 'nui-widgets': '^15.3.2', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'react-bootstrap', 'nui-widgets': '^0.31.5', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'react-datepicker', 'nui-widgets': '^0.60.2', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'react-onclickoutside', 'nui-widgets': '^6.0.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'react-overlays', 'nui-widgets': '^0.7.3', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'react-popper', 'nui-widgets': '^0.8.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'scroll-into-view', 'nui-widgets': '^1.9.1', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'shortid', 'nui-widgets': '^2.2.6', 'ui-assets': 'missing' } +0ms

DevDependencies:
  packcomp:cli diffObj: { Module: '@concur/nui-config', 'nui-widgets': '3.0.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: '@concur/nui-webpack', 'nui-widgets': '^2.0.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: '@concur/ui-assets', 'nui-widgets': '4.11.7', 'ui-assets': 'missing' } +1ms
  packcomp:cli diffObj: { Module: 'babel-cli', 'nui-widgets': '^6.3.17', 'ui-assets': '^6.18.0' } +0ms
  packcomp:cli diffObj: { Module: 'babel-core', 'nui-widgets': '^6.3.26', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'babel-eslint', 'nui-widgets': '^7.0.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'babel-loader', 'nui-widgets': '^7.0.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'babel-minify-webpack-plugin', 'nui-widgets': '^0.2.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'babel-plugin-react-transform', 'nui-widgets': '^2.0.0-beta1', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'babel-polyfill', 'nui-widgets': '^6.23.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'babel-preset-es2015', 'nui-widgets': '^6.3.13', 'ui-assets': '^6.18.0' } +0ms
  packcomp:cli diffObj: { Module: 'babel-preset-react', 'nui-widgets': '^6.3.13', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'babel-preset-react-hmre', 'nui-widgets': '^1.0.1', 'ui-assets': 'missing' } +1ms
  packcomp:cli diffObj: { Module: 'babel-preset-stage-0', 'nui-widgets': '^6.3.13', 'ui-assets': '^6.16.0' } +0ms
  packcomp:cli diffObj: { Module: 'css-loader', 'nui-widgets': '^0.26.1', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'enzyme', 'nui-widgets': '^3.1.1', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'enzyme-adapter-react-16', 'nui-widgets': '^1.1.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'eslint', 'nui-widgets': '^4.18.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'eslint-plugin-compat', 'nui-widgets': '^1.0.4', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'eslint-plugin-jsx-a11y', 'nui-widgets': '^6.0.2', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'eslint-plugin-loosely-restrict-imports', 'nui-widgets': '^0.1.15', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'eslint-plugin-react', 'nui-widgets': '7.7.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'extract-text-webpack-plugin', 'nui-widgets': '^2.1.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'generate-json-webpack-plugin', 'nui-widgets': '^0.2.2', 'ui-assets': 'missing' } +1ms
  packcomp:cli diffObj: { Module: 'html-webpack-plugin', 'nui-widgets': '^2.28.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'jsdom', 'nui-widgets': '^11.6.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'less', 'nui-widgets': '^2.7.2', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'less-loader', 'nui-widgets': '^4.0.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'properties-reader', 'nui-widgets': '0.0.15', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'puppeteer', 'nui-widgets': '^1.0.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'react', 'nui-widgets': '^16.0.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'react-dom', 'nui-widgets': '^16.0.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'react-router', 'nui-widgets': '^3.0.5', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'request-polyfilled-animation-frame', 'nui-widgets': '0.0.3', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'rimraf', 'nui-widgets': '^2.6.1', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'should', 'nui-widgets': '^7.1.1', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'sinon', 'nui-widgets': '^1.17.4', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'size-limit', 'nui-widgets': '^0.18.1', 'ui-assets': 'missing' } +1ms
  packcomp:cli diffObj: { Module: 'style-loader', 'nui-widgets': '^0.16.1', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'unminified-webpack-plugin', 'nui-widgets': '^1.2.0', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'webpack', 'nui-widgets': '^2.3.3', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'webpack-bundle-tracker', 'nui-widgets': '^0.2.1', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: 'webpack-dev-server', 'nui-widgets': '^2.4.5', 'ui-assets': 'missing' } +0ms
  packcomp:cli diffObj: { Module: '@concur/ui-asset-tools', 'nui-widgets': 'missing', 'ui-assets': '^1.1.0-rc.0' } +0ms
  packcomp:cli diffObj: { Module: 'bootstrap', 'nui-widgets': 'missing', 'ui-assets': '3.3.7' } +0ms
  packcomp:cli diffObj: { Module: 'dir-babel', 'nui-widgets': 'missing', 'ui-assets': '^1.0.8' } +1ms
  packcomp:cli diffObj: { Module: 'fs-extra', 'nui-widgets': 'missing', 'ui-assets': '^2.1.2' } +0ms
  packcomp:cli diffObj: { Module: 'linklocal', 'nui-widgets': 'missing', 'ui-assets': '^2.8.0' } +0ms
  packcomp:cli diffObj: { Module: 'tinymce', 'nui-widgets': 'missing', 'ui-assets': '4.5.6' } +0ms
  packcomp:cli diffObj: { Module: 'unzip', 'nui-widgets': 'missing', 'ui-assets': '^0.1.11' } +0ms
```

**Comment by guyellis** [comment]
@bcullman are you running this on Windows and/or WSL?

**Comment by bcullman** [comment]
MacOS, latest

**Comment by guyellis** [comment]
@bcullman I can see that happen on the Mac but not sure why. I've published a new version (`1.0.0.`) and tested it on a Mac and that seems to work. Can you try and install the latest `1.0.0` version and try that?

If you're installing it globally check that you've got the latest version by running `npm list -g packcomp`

**Comment by bcullman** [comment]
thank you! that worked!
